src/redux/categorySlice.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-disable*/
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = [
+  {
+    category: '',
+  },
+];
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    checkCategory: state => ({
+      ...state,
+      category: 'Under construction',
+    }),
```

Comment:
- [x] Kindly make sure on your reducer, the action is also [passed as an argument so in that way you have to set what shall occur when that action is dispatched.
```suggestion
  reducers: {
    checkCategory: (state, action) => ({
      ...state,
      category: 'Under construction',
    }),
```